src/App.tsx

```diff
@@ -97,6 +102,46 @@ const App = () => {
     setShowAppointmentFlow(true);
   }
 
+  useEffect(() => {
+    function triggerRouter() {
```

Comment:
I feel like we actually want to use a routing library longer-term, but if this works 👍